dll/win32/msi/dialog.c

```diff
@@ -3157,8 +3157,17 @@ static void msi_dialog_vcl_add_columns( msi_dialog *dialog, msi_control *control
 
         /* the width must be a positive number
          * if a width is invalid, all remaining columns are hidden
+         * Skip in case of prefix the string of displayed characters with {\style} or {&style}.
```

Comment:
Do not change the code outside of the ifdef block. Either undo it, or move that comment into the ifdef-block

Reply:
Fixed

Reply:
No need for the #else block.
Just move that single comment line into your 2nd #ifdef block and you are done. You don't have to extend the existing comment.

Reply:
Okay, done